src/main/resources/Ideas/IdeasSheet.xml

```diff
@@ -356,9 +372,48 @@
   color: $theme.linkColor;
   cursor: pointer;
   text-decoration: underline;
+}
+
+.recent {
+  line-height: 30px;
+}
+
+.recent .xvote-pro, .recent .xvote-con {
+  margin: .1em .3em .5em 0;
+  width: 15px;
+  height: 15px;
+  line-height: 15px;
+  font-size: 15px;
+  padding: 1px;
+  border-radius: 0;
+  font-size: .8em;
+}
+
+.recent .noitems {
+  font-size: inherit;
```

Comment:
Yes, because IMO this:
![image](https://cloud.githubusercontent.com/assets/8993426/5875238/b752b68e-a313-11e4-97f6-15ce3f1ef1bf.png)
looks better than this: 
![image](https://cloud.githubusercontent.com/assets/8993426/5875241/be78d8e4-a313-11e4-94c2-bb7908cf52da.png)